src/main/java/com/minecolonies/blockout/PaneParams.java

```diff
@@ -346,7 +346,7 @@ private static int parseScalableIntegerRegexMatch(final Matcher m, final int def
 
             if ("%".equals(m.group(2)))
             {
-                value = scale * MathHelper.clamp_int(value, 0, 100) / 100;
+                value = scale * MathHelper.clamp(value, 0, 100) / 100;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 100 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 100 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)